pandas/core/window/ewm.py

```diff
@@ -410,9 +405,21 @@ def cov(
             observations will be used.
         bias : bool, default False
             Use a standard estimation bias correction.
-        **kwargs
-           Keyword arguments to be passed into func.
         """
+        ),
+        numpy_args_kwargs=numpy_args_kwargs,
+        agg_method="cov",
+        other_see_also="",
+        notes="",
+        examples="",
+    )
+    def cov(
+        self,
+        other: Optional[Union[np.ndarray, FrameOrSeries]] = None,
+        pairwise: Optional[bool] = None,
+        bias: bool = False,
+        **kwargs,
+    ):
```

Comment:
no args here.

on master
```
    Parameters
    ----------
    other : Series, DataFrame, or ndarray, optional
        If not supplied then will default to self and produce pairwise
        output.
    pairwise : bool, default None
        If False then only matching columns between self and other will be
        used and the output will be a DataFrame.
        If True then all pairwise combinations will be calculated and the
        output will be a MultiIndex DataFrame in the case of DataFrame
        inputs. In the case of missing elements, only complete pairwise
        observations will be used.
    bias : bool, default False
        Use a standard estimation bias correction.
    **kwargs
       Keyword arguments to be passed into func.
```

this PR
```
    Parameters
    ----------

    other : Series, DataFrame, or ndarray, optional
        If not supplied then will default to self and produce pairwise
        output.
    pairwise : bool, default None
        If False then only matching columns between self and other will be
        used and the output will be a DataFrame.
        If True then all pairwise combinations will be calculated and the
        output will be a MultiIndex DataFrame in the case of DataFrame
        inputs. In the case of missing elements, only complete pairwise
        observations will be used.
    bias : bool, default False
        Use a standard estimation bias correction.

    *args, **kwargs
        For Numpy compatibility and has no effect on the computed value.
```

Reply:
The explanation of `kwargs` is also quite different. The old " Keyword arguments to be passed into func." was not correct I suppose?

Reply:
Yeah `**kwargs` was supposed just exist for compatibility and didn't have an effect on how the result was calculated.